.github/contributing.md

```diff
@@ -94,6 +94,16 @@ And some options:
 - `--verbose=false` to avoid the test list
 - `--coverage=false` to avoid collecting coverage, faster for the part you need the test to pass
 
+### Quick Local CI Check
+
+For a fast (~1-2 min) local check that mirrors CI:
+
+```bash
+pnpm local-ci
+```
+
+This runs all checks in parallel: auto-fixes, lint, type-check, and only the test shards affected by your changes. Use `--no-tests` for lint-only, or `--no-fix` to skip auto-fixing.
```

Comment:
PR description says `--fast` for not tests ?

Reply:
Changed my mind: disable tests with --no-test, disable fix with --no-fix